components/tree/TreeNodeTitle.razor

```diff
@@ -36,9 +36,9 @@
                 {
                     @SelfNode.TitleTemplate
                 }
-                else if (SelfNode.Matched)
+                else if (SelfNode.Matched && !string.IsNullOrWhiteSpace(TreeComponent.SearchValue))
```

Comment:
We can set that if `TreeComponent.SearchValue` is null or white space, then `SelfNode.Matched` is not true.

Reply:
Hi James,

If remove null check with TreeComponent.SearchValue you will have a erro due RenderTreeBuilder for tree is execute before Selfnode.Matched is false. This generate error after this line because using properties from TreeComponent.SearchValue and it's null and code afterwards is @((MarkupString)SelfNode.Title.Replace(TreeComponent.SearchValue, value))
It's not posible replace data in a string null object.

TEST indications:
Using a tree with Draggable and using SearchValue (avoid using TitleTemplate) and a search component (same as demo but add Draggable to Tree component) and ... try write chars in search box , all is ok, but try delete chars even last and error is generate.

regards